migrations-todo.txt

```diff
@@ -0,0 +1,4 @@
+migrations
+
+remove #domain_id from several models
+look for grade_span, grade_span_expection, and domain on all the material models
```

Comment:
Hasn't this been done?  

It seems to me that the columns and associations have all been removed.

But if this hasn't been done I am worried that we are going to forget to do it.

Reply:
Oops yes. I intended to remove that file but forgot. Good catch. New commit deletes it.